flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
why do you disallow this from running on a RaspberryPi?

Reply:
yea, this should be removed